test/jdk/java/lang/Class/getEnclosingClass/EnclosingClassTest.java

```diff
@@ -59,71 +48,172 @@
+        try {
+            Files.deleteIfExists(pkg2File);
+            Files.deleteIfExists(pkg2Dir);
+            Files.deleteIfExists(pkg1File);
+            Files.deleteIfExists(pkg1Dir);
+        } catch (IOException ex) {
+            ex.printStackTrace();
+        }
+    }
+
+    private void createAndWriteEnclosingClasses(final Path source, final Path target, final String packagePath) {
+        try (BufferedReader br = new BufferedReader(new FileReader(source.toFile()));
+        PrintWriter bw = new PrintWriter(new FileWriter(target.toFile()))) {
+            String line;
+            while ((line = br.readLine()) != null) {
+                if (line.contains("canonical=\"EnclosingClass")) {
+                    line = line.replaceAll("canonical=\"EnclosingClass", "canonical=\"" + packagePath + ".EnclosingClass");
+                } else if (line.contains("\"class EnclosingClass")) {
+                    line = line.replaceAll("\"class EnclosingClass", "\" class " + packagePath + ".EnclosingClass");
+                } else if (line.contains("//package")) {
+                    line = line.replaceAll("//package", "package " + packagePath + ";");
+                }
+                bw.println(line);
+            }
+        } catch (IOException ex) {
+            ex.printStackTrace();
+        }
+    }
+
+    private void info(final Class<?> c, final Class<?> encClass, final String desc) {
+        if (!"".equals(desc)) {
             System.out.println(desc + ":");
+        }
         System.out.println(c);
         System.out.println("\tis enclosed by:\t\t" + encClass);
-        System.out.println("\thas simple name:\t`" +
-                           c.getSimpleName() + "'");
-        System.out.println("\thas canonical name:\t`" +
-                           c.getCanonicalName() + "'");
+        System.out.println("\thas simple name:\t`" + c.getSimpleName() + "'");
+        System.out.println("\thas canonical name:\t`" + c.getCanonicalName() + "'");
     }
 
-    static void match(String actual, String expected) {
-        assert((actual == null && expected == null) || actual.equals(expected));
-        System.out.println("\t`" +
-                           actual + "' matches expected `" +
-                           expected + "'");
+    private void match(final String actual, final String expected) {
+        System.out.println("actual:" + actual + "expected:" + expected);
+        assert ((actual == null && expected == null) || actual.trim().equals(expected.trim()));
+        System.out.println("\t`" + actual + "' matches expected `" + expected + "'");
     }
 
-    static void check(Class<?> c, Class<?> enc,
-                      String encName, String encNameExpected,
-                      String simpleName, String simpleNameExpected,
-                      String canonicalName, String canonicalNameExpected) {
+    private void check(final Class<?> c, final Class<?> enc,
```

Comment:
I see that you make all parameters in all methods final.   It just adds noise.   Can you leave it out?

Reply:
To avoid the Checkstyle warnings, I added them.

Reply:
Is it from your IDE configurations?  You can turn off Checkstyle warnings.  This just adds noise.

Reply:
I also assume some of the formatting changes are changed by your IDE suggestion.  Can you please revert the formatting changes (there are quite a few of them).   This will help the reviewers.